packages/mobile/src/navigator/Navigator.tsx

```diff
@@ -592,25 +600,56 @@ export function MainStackScreen() {
       pincodeType,
       account,
       hasSeenVerificationNux,
+      shouldSkipOnboardingEducationScreen,
     } = mapStateToProps(store.getState())
 
+    // Remove Onboarding Education Screen Experiment: Because remote configs are fetched after the initial route is launched,
```

Comment:
Can we use a remote config and pull it in early like we did with sentry? If it's a random 50 / 50 hardcoded split e2e tests will fail 50% of the time unless they are adjusted.

Reply:
The reason was fetching from firebase takes extra time, i estimated it to be about .8s so didn't want to prolong the initial loading time. It's possible that by doing some sort of parallel request, the actual time added to initial loading would be a lot less than .8s. Also another reason is just want to get a quick exp out of door without spending a lot of time on it. 

That was a good point about e2e testing. I've updated the e2e test and hopefully it will pass